### PR TITLE
Using gradle wrapper 3.0-all, added debug-in-intellij instructions

### PR DESCRIPTION
**Pull request by zsimic**
Updated wrapper by running `./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-3.0-all.zip`
The `-all` variant is useful for debugging.

Added hopefully clear instructions on how to relatively conveniently debug in IntelliJ

### CODE REVIEW

**Comment by sholsapp** [comment]
Hey Zoran! :)

Code changes LGTM, but Ethan you're probably more familiar with the `gradlew` bits. Can you check?

File: gradlew [hunk 1]
```diff
@@ -161,4 +161,9 @@ function splitJvmOpts() {
 eval splitJvmOpts $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS
 JVM_OPTS[${#JVM_OPTS[*]}]="-Dorg.gradle.appname=$APP_BASE_NAME"
 
+# by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
```
Comment (ethankhall) [inline gradlew]: Did you do this change or was it scripted?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Thu Aug 25 07:56:28 PDT 2016
+#Sun Feb 26 21:35:32 PST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.0-all.zip
```
Comment (ethankhall) [inline gradle/wrapper/gradle-wrapper.properties]: Might as well bump to 3.4-all while were at it.

File: docs/developers.md [hunk 3]
```diff
@@ -54,3 +54,59 @@ We choose to validate releases of PyGradle internally before releasing them
 publicly. Therefore, if you want a release, please reach out to us so that we
 can do this if you're blocked on getting a new release out. As with
 contributions, please be patient while we do this.
+
+
+## Debugging in IntelliJ
```
Comment (ethankhall) [inline docs/developers.md]: /shrug I use the CLI to do it. Makes things way simpler.

File: gradlew (hunk 1 above)
Comment (zsimic) [inline gradlew]: I didn't change this, the command I ran did it: `./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-3.0-all.zip`

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (zsimic) [inline gradle/wrapper/gradle-wrapper.properties]: I thought about that, but wasn't sure if you wanted to stick with 3.0 to make sure pygradle doesn't start requiring a later version of gradle...

For debugging, any version with the `-all` flavor will do.

File: docs/developers.md (hunk 3 above)
Comment (zsimic) [inline docs/developers.md]: Debugger is very useful when you're new to the code :)

File: gradlew (hunk 1 above)
Comment (ethankhall) [inline gradlew]: Ok cool. Making sure that it wasn't manual.

**Pull request merged**